### DISCUSSION TITLE
OS usage by OCCA applications

### DISCUSSION THREAD

**Discussion by kris-rowe** [libocca/occa#755] [Polls]
OCCA is well supported and tested on several Linux distributions; however, the same is not necessarily true for macOS and Windows.
Understanding the needs of downstream project will help determine how best to focus future development efforts.

**Comment by nnnunnn** [comment]
I run multi-gpu occa apps on native Windows 11 using Visual Studio 22. If there is any interest, I am keen to incorporate my patches.
Nigel

**Comment by tcew** [comment]
I would like to be using macOS, but getting things to run is a bit of a chore
